src/engines/witcher/witcher.cpp

```diff
@@ -184,6 +186,9 @@ void WitcherEngine::init() {
 	progress.step("Initializing internal game config");
 	initGameConfig();
 
+	progress.step("Initializing Lua subsystem");
```

Comment:
If you add a load progress step, you need to increase the total number of steps here: https://github.com/smbas/xoreos/blob/7edbfaadfc814197f469f35e849ef66c8a9518dd/src/engines/witcher/witcher.cpp#L162 . Yeah, that's a bit wonky, I know.